packages/angular/projects/angular/src/components/radio-button/radio-button.strategies.ts

```diff
@@ -50,12 +43,25 @@ function createRadioGroupStrategy(
 
   const setChecked = () => {};
 
+  const onInit = () => {
+    radioGroupCore$.pipe(waitContextAndDo())
```

Comment:
Suggestion: Looks like radioGroupCore has already has context value. May be we should simplify code and remove waitContextAndDo at all.

Reply:
The core has context here only if we still init it in in the ngOnInit of the radio-group component.

Without helper operator ```waitContextAndDo``` it looks like:
```ts
radioGroupCore$.pipe(
  filter((context): context is RadioGroupCore<RadioGroupValue> => !!context),
  take(1),
).subscribe(...);
```

If we remove the filter, we win just one line, but lose in safety and future changes:
```ts
radioGroupCore$.pipe(
  take(1),
).subscribe(...);
```

So, let's leave it as is :)

Reply:
I suppose without waitContextAndDo it's simplified to

```
radioGroupCore$.subscribe(...);
```